### PR TITLE
4.next: added command description doc

### PR DESCRIPTION
**Pull request by LordSimal**
Ref: https://github.com/cakephp/cakephp/pull/16366

### CODE REVIEW

File: en/console-commands/commands.rst [hunk 1]
```diff
@@ -240,6 +240,35 @@ You may need to call other commands from your command. You can use
...
+
+    class UserCommand extends Command
+    {
+        public static function getDescription(): string
+        {
+            return 'My custom description';
+        }
+    }
+
+This will show your description in the Cake CLI::
```
Comment (othercorey) [inline en/console-commands/commands.rst]: This needs to use a shell code block

File: en/console-commands/commands.rst [hunk 2]
```diff
@@ -240,6 +240,35 @@ You may need to call other commands from your command. You can use
...
+
+    bin/cake
+
+    App:
+      - user
+      └─── My custom description
+
+As well as in the help section of your command::
+
+    cake user --help
```
Comment (othercorey) [inline en/console-commands/commands.rst:270]: This needs a shell code block too.

File: en/console-commands/commands.rst (hunk 1 above)
Comment (markstory) [reply en/console-commands/commands.rst]:
```rst
... code-block:: console 

    bin/cake

```
is what you want.

**Review by Arhell — APPROVED** [review]

File: en/console-commands/commands.rst (hunk 1 above)
Comment (LordSimal) [reply en/console-commands/commands.rst]: I first thought you meant adding a separte doc for CakePHP Shells, not the RST "shell" code styling for that block 😅 thanks markstory 😄

File: en/console-commands/commands.rst (hunk 2 above)
Comment (LordSimal) [reply en/console-commands/commands.rst:270]: done!

File: en/console-commands/commands.rst [hunk 3]
```diff
@@ -255,6 +255,7 @@ You may want to set a command description via::
 
 This will show your description in the Cake CLI::
 
+... code-block:: console
```
Comment (markstory) [inline en/console-commands/commands.rst]:
```suggestion
... code-block:: console

```

pretty sure you need the newline here.

File: en/console-commands/commands.rst [hunk 4]
```diff
@@ -263,6 +264,7 @@ This will show your description in the Cake CLI::
 
 As well as in the help section of your command::
 
+... code-block:: console
```
Comment (markstory) [inline en/console-commands/commands.rst]:
```suggestion
... code-block:: console

```

File: en/console-commands/commands.rst (hunk 3 above)
Comment (LordSimal) [reply en/console-commands/commands.rst]: god damn I hate RST 🙈 😂

File: en/console-commands/commands.rst (hunk 1 above)
Comment (othercorey) [reply en/console-commands/commands.rst]: it's supposed to be two . not three. `.. code-block::`.

https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-code-block

File: en/console-commands/commands.rst (hunk 3 above)
Comment (markstory) [reply en/console-commands/commands.rst]: It does take a bit of learning. I do like how it enforces a consistent convention around whitespace usage in the source. That kind of document formatting can be unwieldy in markdown.

**Pull request merged**